cmd/crio/main.go

```diff
@@ -55,6 +55,9 @@ func validateRuntimeConfig(config *server.Config) error {
 	if ok && config.RuntimeConfig.RuntimeUntrustedWorkload != "" {
 		return fmt.Errorf("conflicting definitions: configuration includes runtime_untrusted_workload and runtimes[%q]", oci.UntrustedRuntime)
 	}
+	if config.RuntimeConfig.Runtime != "" {
```

Comment:
Please move this to the top of this function with the block of `if` statements responsible for checking all the deprecated use cases. A comment would be nice to specify that we're checking all the deprecated behaviors first.